contributing_to_docs/create_or_edit_content.adoc

```diff
@@ -0,0 +1,158 @@
+= Create new content or edit existing content
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Create working branch to contribute new content or updates
+
+toc::[]
+
+== Before you begin
+Before you proceed, we recommend that you have completed the following:
+
+* Read and reviewed the link:contributing.html[Contribute to OpenShift documentation] topic to understand some basics
+* link:tools_and_setup.html[Installed and set up the tools and software] required to contribute
+* Read and reviewed the link:doc_guidelines.html[Documentation guidelines] topic to understand the basic guidelines for consistency
+
+== Ensure your local repository is in sync with the remote
+Before you create a local working branch, it is good practice to ensure that your local `master` branch is in sync with the remote and that you have all the latest changes. You must also ensure that your forked repository is also in sync with the remote repository.
+
+1. From your local repository, make sure you have the `master` branch checked out:
++
+----
+$ git checkout master
+----
```

Comment:
@CowboysFan Sorry, I shouldn't have said repo. I meant branch. So instead of saying "make sure you have the 'master' branch checked out" I feel it should say 'make sure you are in the 'master' branch"